django_docker_engine/proxy.py

```diff
@@ -70,10 +70,16 @@ def url_patterns(self):
             )
         ]
 
+    def _get_proxy_view(self, container_url):
```

Comment:
I'm going to inline this: If we kept it, we'd have `_get_proxy_view`, `_internal_proxy_view`, and `_proxy_view`